src/test/java/school/lemon/changerequest/java/generics/container/ContainerTest.java

```diff
@@ -0,0 +1,91 @@
+package school.lemon.changerequest.java.generics.container;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+/**
+ * Created by User on 03.01.2017.
+ */
+public class ContainerTest {
+    Container container = new Container();
```

Comment:
You should create `@Before` fixture and re-initialize container before each test to be sure, that there won't be any side-effects (like you'll add value in one test and it will be in available in other test)